addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
     type='text'
-    value={{value}}
-    placeholder={{placeholder}}
-    oninput={{action 'input'}}
-    onkeydown={{action 'keydown'}}
-  />
+    spellcheck=false
+    value=value
+    placeholder=placeholder
+    input=(action 'input')
+    keyDown=(action 'keydown')
+  }}
 
-  <button title='clear input' class='clear' disabled={{valueIsEmpty}} {{action 'clear'}}>&#10005;</button>
+  {{#if showReset}}
+    <i title='clear input' class='clear' {{action 'clear'}}>
```

Comment:
Yes and no. I like the idea of the button being easily configurable so that it can match the styling of the consuming app, but I don't like embedding HTML/SVG in my JS. I know that's very React-ish, but it's not very Ember-ish.

I think it's pretty common to overload the `i` tag for use as clickable icons, but if you want to go with a button for sake of semantics, I won't argue the point. As long as I can remove the button's styling to make it as clean as possible, I 'm good with that.